Python/other/Unique_number_III.py

```diff
@@ -0,0 +1,37 @@
+# Unique_Number_III
+# Given an array that contains all elements occurring 3 times, but one
+# occurs only once. Find that unique element.
+
+
+def uniquee_element(array, size):
+    for i in range(size):
+        j = 0
+        while(j < size):
```

Comment:
so I can see a `while` loop under `if` then how is the time complexity `O(size)` here?

Reply:
> so I can see a `while` loop under `for` loop then how is the time complexity `O(size)` here?

Sorry ma'am ,time complexity will be O(size*size),because while loop is under for loop